desktop/apps/artsy_primer/personalize/client/views/price_range.coffee

```diff
@@ -4,12 +4,20 @@ createSlider = require '../../../../../components/slider/index.coffee'
 template = -> require('../../templates/price_range.jade') arguments...
 
 module.exports = class PriceRangeView extends StepView
+
+  events: 
+    'keyup #anything-else' : 'updateNotes'
+
   formatter: (val, index) ->
     if index is 0
       "#{formatMoney(val, { precision: 0 })}"
     else
       "#{formatMoney(val, { symbol: "", precision: 0 })}"
 
+  updateNotes: ->
+    val = @$('#anything-else').val()
+    @user.set notes: val
```

Comment:
I guess this is cool to use, but this field on the `User` model is super-old, there's....maybe a better place for this? I'm not sure though.

Either way, the field isn't part of the [user export to looker](https://github.com/artsy/gravity/blob/b7eaeb54a6f7a3a23d2a544f2e8f68d5e446e79f/app/reports/users_report.rb) and so isn't accessible there, but it does seem accessible from within Torque. Not sure if it's really accessible anywhere else?